### ISSUE TITLE
How to generate data/long_arg_recombinants.csv

### ISSUE THREAD

**Issue by hyanwong** [jeromekelleher/sc2ts-paper#208]
We seem to have lost any code that creates this file, and we can't use the existing one for the new Viridian data. It is required for plotting. I think its just the output of the TreeInfo command.

It looks like the change that created a ref to the missing file was here: https://github.com/jeromekelleher/sc2ts-paper/commit/7b2285d448bf4a2e563035820248439ef698cf9f#diff-04a5013aee36a31513cee04eb4b37ab3488e98d1f8cf40203b78b2cdbb797bdf

**Comment by jeromekelleher** [comment]
It's the output of the ``TreeInfo.recombinant_summary()`` function

**Comment by hyanwong** [comment]
Yep. I guess we should add something that generates that to the Makefile? Rather than doing it in plot.py?

**Comment by jeromekelleher** [comment]
Yeah. Let's hold off on that level of automation until the sc2ts APIs have finalised.

**Comment by jeromekelleher** [comment]
So, the idea is that the sc2ts repo has the inference and basic stats generation code, which we release on pypi. This repo has the code for running it on the viridian dataset, and the code for all the fancy stuff (which would be hard to maintain, keep working properly etc)
